specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/settings.json

```diff
@@ -170,19 +169,13 @@
             "description": "The value of the pool setting."
           },
           "type": {
-            "type": {
               "enum": [
                 "boolean"
               ],
             "modelAsString": true,
```

Comment:
This has to go into an `x-ms-enum`. See https://github.com/Azure/autorest/tree/main/docs/extensions#x-ms-enum